nengo_viz/static/viz_value.js

```diff
@@ -22,28 +23,50 @@ VIZ.Value = function(args) {
     this.svg = d3.select(this.div).append('svg')
         .attr('width', '100%')
         .attr('height', '100%');
-        
+
+    //Add the title 
+    this.svg.append("text")
+        .text(args.id)
```

Comment:
(note: to get at this value, it'll have to be passed in from the server.  I think the easiest way is to do that on the ```Value.javascript()``` call.)